README.md

```diff
@@ -2,26 +2,21 @@
 
 # Project Name
 
-> Description the project.
-
+> Hello Microverse
```

Comment:
- [ ] Please add the project's actual name "Hello Microverse" after the `#` and kindly add the project description after `>` to make your Readme file look more professional.